test/unit/accept_patient_reports.js

```diff
@@ -54,27 +57,57 @@ exports['onMatch callback empty if form not included'] = function(test) {
 exports['onMatch with matching form calls getRegistrations and then matchRegistrations'] = function(test) {
 
     var getRegistrations,
+        getPatientContactUuid,
         matchRegistrations;
 
     sinon.stub(transition, 'getAcceptedReports').returns([ { form: 'x' }, { form: 'z' } ]);
 
-    getRegistrations = sinon.stub(utils, 'getRegistrations').callsArgWithAsync(1, null, []);
-    matchRegistrations = sinon.stub(transition, 'matchRegistrations').callsArgWithAsync(1, null, true);
+    getRegistrations = sinon.stub(utils, 'getRegistrations').callsArgWith(1, null, []);
+    getPatientContactUuid = sinon.stub(utils, 'getPatientContactUuid').callsArgWith(2);
+    matchRegistrations = sinon.stub(transition, 'matchRegistrations').callsArgWith(1, null, true);
 
     transition.onMatch({
         doc: {
-            form: 'x'
+            form: 'x',
+            fields: { patient_id: 'x' }
         }
     }, {}, {}, function(err, complete) {
         test.equals(complete, true);
 
         test.equals(getRegistrations.called, true);
+        test.equals(getPatientContactUuid.called, true);
```

Comment:
Fair call. I'll expand it out. I think I kept it minimal because I felt it was tested more elsewhere? IDK, a lot of sentinel tests seem to test similar areas. Confusing.